intf_.mod

```diff
@@ -677,11 +681,11 @@ PROCEDURE trvsp ()
 {
   VERBATIM 
   int i, flag; 
-  double ind, t0;
```

Comment:
`t0` is actually a macro in the translated code, and there are warnings about out-of-range access. Presumably the intent was to use a local variable.

Reply:
Just to add the context: MOD file has `t0` as the RANGE variable and the user also declares `t0` variable within  verbatim:


```cpp
 NEURON {
   RANGE t0,
   ...
}

 PROCEDURE trvsp ()
 {
   VERBATIM
   int i, flag;
   double ind, t0_local;
   ip=IDP;
   flag=(int) *getarg(1);
   if (subsvint==0.) {printf("trvsp"); return(0.);}
   ind=isp[0]; t0=vsp[0];
   if (flag==1) {
     for (i=0; i<vspn; i++) {
       if (isp[i]!=ind) {
         vsp[i-1]=1.e9;
         ind=isp[i];
       }
     }
     vsp[vspn-1]=1.e9;
   } else if (flag==2) {
     for (i=0; i<vspn; i++) {
       if (isp[i]!=ind) {
         vsp[i-1] = t0 + subsvint;
         ind = isp[i];
         t0 = vsp[i];
       }
     }
     vsp[vspn-1] = t0 + subsvint;
   } else {printf("trvsp flag %d not recognized\n",flag); hxe();}
   ENDVERBATIM
 }
```

and this was resulting into warnings like:

```console
intf_.cpp:1325:15: warning: array index 26 is past the end of the array (which contains 26 elements) [-Warray-bounds]
  ind=isp[0]; t0=vsp[0];
              ^~
intf_.cpp:131:12: note: expanded from macro 't0'
#define t0 _p[26]
           ^  ~~
intf_.cpp:1321:3: note: array '_p' declared here
  double ind, t0;
  ^
intf_.cpp:1337:20: warning: array index 26 is past the end of the array (which contains 26 elements) [-Warray-bounds]
        vsp[i-1] = t0 + subsvint;
                   ^~
intf_.cpp:131:12: note: expanded from macro 't0'
#define t0 _p[26]
           ^  ~~
intf_.cpp:1321:3: note: array '_p' declared here
  double ind, t0;
  ^
intf_.cpp:1339:9: warning: array index 26 is past the end of the array (which contains 26 elements) [-Warray-bounds]
        t0 = vsp[i];
        ^~
intf_.cpp:131:12: note: expanded from macro 't0'
#define t0 _p[26]
           ^  ~~
intf_.cpp:1321:3: note: array '_p' declared here
  double ind, t0;
  ^
intf_.cpp:1342:19: warning: array index 26 is past the end of the array (which contains 26 elements) [-Warray-bounds]
    vsp[vspn-1] = t0 + subsvint;
                  ^~
intf_.cpp:131:12: note: expanded from macro 't0'
#define t0 _p[26]
           ^  ~~
intf_.cpp:1321:3: note: array '_p' declared here
  double ind, t0;
  ^
```

So definitely `t0` can't be declared as a local variable under verbatim. Make sense to rename that.